public/app/features/dashboard/components/HelpWizard/SupportSnapshotService.ts

```diff
@@ -82,7 +82,7 @@ export class SupportSnapshotService extends StateManagerBase<SupportSnapshotStat
 
     if (!panel.isAngularPlugin()) {
       try {
-        const oldModel = new DashboardModel(snapshot);
+        const oldModel = new DashboardModel(snapshot, { isEmbedded: true });
```

Comment:
Why's that `isEmbedded` meta passed here, is it used anywhere?

Reply:
@dprokop yes, when we create the DashboardScene inside createDashboardSceneFromDashboardModel we take the meta object from DashboardModel.meta